### ISSUE TITLE
How build this app locally?

### ISSUE THREAD

**Issue by dodo325** [soler91/noita-together#61]
I am trying to build this project on linux (Ubuntu 20.04). Please let me know which dependencies to install and which build commands?

I am trying `npm install`, `e build` (with `@electron/build-tools@1.0.4`) but it didn't work.

Help!

**Comment by dodo325** [comment]
This helps to me: `npm run electron:serve`

**Comment by dodo325** [comment]
`Login with Twitch.tv` button rederect to `https://undefined/auth/login`

**Comment by VacuusUmbra** [comment]
I'm trying to build it locally but  this
> This helps to me: `npm run electron:serve`

is returning ERR_INVALID_URL for dist_electron. 

Were you able to get this to run locally?

**Comment by stefnotch** [comment]
> `Login with Twitch.tv` button rederect to `https://undefined/auth/login`

Yes, this app assumes that there is a `.env` file with the "correct" contents. You could join the Discord server, I suppose https://discord.com/invite/ex3XFkXCgM

Alternatively, you can try out the code on my fork https://github.com/stefnotch/noita-together/tree/feat-vue3 (make sure to select the `feat-vue3` branch)
